website/src/contribute/testing.md

```diff
@@ -3,428 +3,18 @@ layout: section
 title: 'Beam Testing'
 section_menu: section-menu/contribute.html
 permalink: /contribute/testing/
+redirect_from:
+  - /contribute/testing/
```

Comment:
I don't think you need this. The use of `redirect_from` is to specify a separate URL from which to bring the user to this page.

Reply:
Done, rm'd and confirmed that it works without redirect_from